site/content/docs/main/data-movement-node-selection.md

```diff
@@ -117,9 +119,7 @@ This example demonstrates how to use both `matchLabels` and `matchExpressions` i
 }
 ```
 
-This sample demonstrates how the `loadAffinity` elements with `StorageClass` field and without `StorageClass` field setting work together. If the VGDP mounting volume is created from StorageClass `kibishii-storage-class`, its pod will run Linux nodes.
-
-The other VGDP instances will run on nodes, which instance type is `Standard_B4ms`.
+**Warning**: In the above example, only the first element (selecting nodes with instance type `Standard_B4ms`) will be applied. The second element with the Linux OS selector and storage class specification will be ignored due to the current limitation where only the first array element is processed.
```

Comment:
I don't think we need this one, for each of the example, as we have already clarified it as an overall statement.